manifest.sh

```diff
@@ -0,0 +1,66 @@
+# Manifest for image directories creation
+# every dest and link_name path will be prefixed by $DEST_DIR/$version
+ 
+DEST_DIR='' # optional, defaults to $PWD
+
+# Files containing distgen directives
+DISTGEN_RULES="
+    src=templates/Dockerfile
+    dest=Dockerfile;
+
+    src=templates/root/usr/share/container-scripts/postgresql/README.md
+    dest=root/usr/share/container-scripts/postgresql/README.md;
+
+    src=templates/root/usr/share/container-scripts/postgresql/common.sh
+    dest=root/usr/share/container-scripts/postgresql/common.sh
+    mode=0755
+"
+
+# Symbolic links
+SYMLINK_RULES="
+    link_target=root/usr/share/container-scripts/postgresql/README.md
+    link_name=README.md;
+
+    link_target=../test
+    link_name=test
+"
```

Comment:
This seems to create absolute symlinks on the host system during build.  Such simlinks - if copied into image - will lead to non-existing paths.